### DISCUSSION TITLE
Flexible buttons in html-button-response

### DISCUSSION THREAD

**Discussion by drordotan** [jspsych/jsPsych#2477] [Ideas]
I propose adding to html-button-response the ability to define a CSS style for each button separately.

I needed this to set a different color for each response button.

My solution was to patch the code by adding to html-button-response a "button_class" parameter, which is an array of style names (one per button).

Attached is the plugin with this update, so you can add it if you want (see lines 115-118). This is how it can be used:

element = {
                type: jsPsychHtmlButtonResponse,
                stimulus: ".....",
                choices: [list of buttons],
                button_class: ['class-1', 'class-2', 'class-3'],
                button_html: '<button class="%custom_class%"/>'
            }

[plugin-html-button-response.txt](https://github.com/jspsych/jsPsych/files/7933766/plugin-html-button-response.txt)

**Comment by jodeleeuw** [comment]
Thanks @drordotan ! This is technically possible with the current plugin because the `button_html` can be an array with distinct HTML for each button. But this is much simpler when the only desired change is the class name.

**Comment by drordotan** [comment]
Thanks, I didn't know it!
